python/sglang/srt/lora/lora_manager.py

```diff
@@ -104,12 +104,18 @@ def init_loras(self):
             self.origin_target_modules = set(self.origin_target_modules) | set(
                 self.configs[name].target_modules
             )
-        self.target_modules = set(
-            [
+        if hasattr(self.base_model, "get_module_name"):
+            self.target_modules = {
                 self.base_model.get_module_name(module)
                 for module in self.origin_target_modules
-            ]
-        )
+            }
+        else:
+            named_modules_dict = dict(self.base_model.named_modules())
+            self.target_modules = {
+                name
+                for name, module in named_modules_dict.items()
+                if module in self.origin_target_modules
```

Comment:
This does not seem correct. `self.origin_target_modules` is a list of strings, not modules. You can refer to the `get_module_name` function in `llama.py`.

Reply:
Fix module name comparison for origin_target_modules in base model. This fix resolves errors encountered in models like Qwen2.5-14B Instruct LoRA, which do not have the get_module_name function.

Reply:
Updated.